src/docs/product/data-management-settings/dynamic-sampling/index.mdx

```diff
@@ -43,6 +44,10 @@ While prioritizing data from the latest release, we also take into consideration
 
 We prioritize data from development environments because these environments typically generate a relatively small number of transactions compared to your production environments. As a result, to gain meaningful insights during your testing phase, we prioritize transaction data from these environments.
 
+### Key Transactions
+
+We prioritize my Key Transactions to 5x times from blended rate. This rule applies to single transactions.
```

Comment:
I have a few questions: 

1. Do you mean _5x times_ or _5 times_? 
2. What do we mean by "blended rate"? Is this something users should already know?
3. Why do we mean by this applying to single transactions. Maybe this is obvious to a developer, but I don't understand what people might think it would apply to instead? An entire trace? 

```suggestion
We prioritize key, or starred, transactions up to five times over the blended rate. This rule applies to single transactions.
```

Reply:
yeah i agree let me rework it. Thanks for review!

Reply:
I would personally phrase it similarly to the other descriptions. Remember that as @jan-auer correctly says, we always need to think about the problem we are solving for customers, therefore I would add to the description the rationale behind this feature.